common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -128,6 +128,7 @@ sealed trait WorkState {
 
   val sourceIdentifier: SourceIdentifier
   val numberOfSources: Int
+  val modifiedTime: Instant
```

Comment:
Is this being used anywhere?

Reply:
yes, it's used [here](https://github.com/wellcomecollection/catalogue/blob/ca5cbbae8835829db436c2c73b5390c3478c7ea0/common/pipeline_storage/src/main/scala/uk/ac/wellcome/pipeline_storage/Indexer.scala#L92) to avoid pattern matching. It's in all the states already anyway

Reply:
Oh yeah, got confused 🙃